### DISCUSSION TITLE
Feature request: Fork a repository from GitHub mobile app

### DISCUSSION THREAD

**Discussion by sanketshah19** [community/community#4401] [Mobile]
It would be very helpful to be able to fork a repository from GitHub mobile app.

**Comment by Zerotask** [comment]
You should update the title with your specific feature request.

**Comment by TheRakeshPurohit** [comment]
I am also looking forward to such a great feature.

**Comment by jsoref** [comment]
https://github.com/orgs/community/discussions/3333 asked for this.

https://twitter.com/github/status/1866883185742622746 claims the feature exists
